src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,32 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [].sort((b, a) => b.health - a.health);
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  hidden = false;
+
+  hide() {
+    this.hidden = !this.hidden;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  bite(animal) {
+    if (animal.hidden === false) {
+      animal.health -= 50;
+    }
+
+    if (animal.health === 0) {
+      Animal.alive.pop();
+    }
```

Comment:
```suggestion
    if (animal.health === 0) {
      Animal.alive.pop();
    }
```
Instead of using two methods (sort and pop), you can just use `filter` and filter all animals whos health > 0